src/main/java/org/jboss/xavier/integrations/jpa/service/AnalysisService.java

```diff
@@ -93,6 +99,7 @@ public void setWorkloadSummaryReportModelAndUpdateStatus(WorkloadSummaryReportMo
 
     public Page<AnalysisModel> findAllByOwner(String owner, int page, int size)
     {
+        assertTrue(size <= paginationLimitMax);
```

Comment:
We can reuse the logic from here https://github.com/project-xavier/xavier-integration/blob/e4293da7eac386a3a82f5ad611a851aebce25daf/src/main/java/org/jboss/xavier/integrations/route/ToBeanRouter.java#L48

Reply:
Well in fact no.... it's the route to-pagebean who takes care of it.
I have already modified the code adding you latest changes about pagination, to this PR